iXBRLViewerPlugin/viewer/src/js/aspect.js

```diff
@@ -74,19 +75,11 @@ Aspect.prototype.valueLabel = function(rolePrefix) {
     }
     else if (this._aspect == 'u') {
         if (this._value === null) {
-            return "<NOUNIT>";
+            return i18next.t("factDetails.noUnit");
         }
         var qname = this._report.qname(this._value);
         if (qname.namespace == "http://www.xbrl.org/2003/iso4217") {
-            if (qname.localname == 'GBP') {
-                return "£";
-            }
-            else if (qname.localname == 'USD') {
-                return "US $";
-            }
-            else if (qname.localname == 'EUR') {
-                return "€";
-            }
+            return i18next.t(`currencies.unitFormat${qname.localname}`, {defaultValue: qname.localname + ' '});
```

Comment:
This change seems odd.  Do different countries actually display the currency symbol differently?

Reply:
I'm not sure (@huizarmx may know otherwise?).  It's possible that some locales may use a symbol where we'd prefer to use the ISO code because the symbol may unfamiliar.

In any event, I think this is a reasonable approach as we've moved a 3-way if block into a single line lookup in a separate JSON file, which is already being used for translating other parts of monetary/numeric amounts (e.g. translating "cents" or "millions").

Reply:
I've seen cases where the user would want the currency symbol on the right side. I think is a simple solution and brings flexibility for displaying different currencies.

Reply:
Ok, thanks.

However, since the `unitFormat` in `es` are currently the same as `en`, does `fallbackLang` allow us to remove all of those so that we don't have to duplicate them in every translation?

Reply:
Of course, that's very reasonable.

Reply:
Now done in 6fa928b